agu-app/src/components/Footer.js

```diff
@@ -0,0 +1,43 @@
+import logob from '../images/therd.png'
+import logoc from '../images/fourth.png'
+import logod from '../images/fif.png'
+import logoe from '../images/sis.png'
+
+
+function Footer() {
+    return(
+
+        <>
+        <div id='footer'>
+        
+        <div className='ending'>
+            
+            <img src={logob}  />
```

Comment:
Usually is a good practice to add the alt attribute for the images as that would increase the usability of our app. Also avoid logoa, logob, logoc, logod 
```suggestion
            <img src={logob} alt="Computer Logo"  />
```